### PR TITLE
Add marimo word lists

### PR DESCRIPTION
**Pull request by kbenoit**
Solves #22 

@koheiw please update the documentation and see how we should handle the return object, as described in #22.

### CODE REVIEW

**Review by koheiw — APPROVED** [review]
Looks great.

**Comment by koheiw** [comment]
I might add Chinese to the collection very soon, because there is a PR.
https://github.com/koheiw/marimo/pull/1

File: R/data.r [hunk 1]
```diff
@@ -71,16 +71,17 @@
...
-#' These are multi-level lists, in the original data.  If you wish to
-#' use them as lists, please access the data object directly.
+#' These are multi-level lists, in the original data.  If you wish to use them
+#' as lists, please access the data object directly.
 #' @usage NULL
-#' @section Usage:
-#' `stopwords(language = "en", source = "marimo")`
-#' @source
-#'   KOHEI TO ADD
+#' @section Usage: `stopwords(language = "en", source = "marimo")`
```
Comment (kbenoit) [inline R/data.r:80]: @koheiw Would it be worth adding more detail about the structure of the nested list components? Also what about naming each level? (as per https://github.com/quanteda/stopwords/issues/22#issuecomment-604380426)

Comment (koheiw) [reply R/data.r:80]: OK, I will add names to `number` and `time` first.

Comment (kbenoit) [reply R/data.r:80]: Great. I had to modify the input yml files slightly to make them readable - I don't know what exactly this was, but it was in the .ja and .he files using `yaml::read_yaml()` - so I suggest you either test the input files with that function in your repo, or modify the source files directly in https://github.com/quanteda/stopwords/tree/add-marimo/data-raw/marimo with the names.

**Pull request merged**